api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Should this be under Apache License? Iam asking since the repo itself is initialized with AGPL.

https://github.com/os-observability/tempo-operator/blob/b45ee30f4107891290c0881960b819cc4a7f2c8f/LICENSE#L1-L6

Reply:
Ahh wait, whats the state here?